readtifimage.py

```diff
@@ -0,0 +1,37 @@
+#Read TIF image
+
+import rasterio
+
+
+def readimage(filepath):
+   """Read TIF image from file.
+
+    Inputs:
+    filepath = path of TIF image file
+
+    Returns:
+    imagearray = image object as numpy array
+    crs = coordinate reference system
+    width = width of digital number array values
+    height = height of digital number array values
+    bands = total number of bands in image
+    bounds = spatial extent of the image mapped on the earth's surface
+
+    :param filename: str
+    :return imagearray: numpy.ndarray
+    :return crs: rasterio.crs.CRS
+    :return width: int
+    :return height: int
+    :return bands: int
+    :return bounds: rasterio.coords.BoundingBox
+    """
+   img = rasterio.open(filepath)
+   inputarray = img.read()
+   crs = img.crs
+   width = img.width
+   height = img.height
+   bands = img.count
+   bounds = img.bounds
+   
+   return inputarray, crs, width, height, bands, bounds
```

Comment:
We should aim to return only one variable, but it is a little complicated. For now, I would suggest we package the data into a PlantCV `Spectral_data` class object, which can store the `inputarray`, `height`, `width`, and `bands`. It does not have attributes for `crs` or `bounds` however. But downstream I am not sure if we would be using these attributes yet, so it might be alright to drop them for the time being.

Longer term we might need a new data class for geospatial data

Reply:
I started working on the implementation of using `Spectral_data` as the output for this function, and left many attributes as None or "unknown". I remember discussing methods to default to the 5 band multispec that we have currently, but what additional data about these bands do we have? 

And like coding wise, does that mean a `mode` input variable for image type where it's a dictionary of possible available bands?

Reply:
Update function name so that it's not the same as an existing plantcv function, confusing to have overlap e.g. `read_geotif`. Assume that HSI will be in ENVI, so this data will be for multispec. grayscale, rgb data mainly. 

Mode: grayscale, rgb & bgr, and multispec (assume 4 band-->planet labs; or 5 band-->altum) AND later wavelength dictionary input?

Reply:
I started to incorporate `_make_pseudo_rgb` and started to fill in a mode -> wavelength-dictionary dictionary. Had the thought of like what type of test data we can provide for each of the preliminary "mode" options, and how we probably want to make the test data as small as possible.

Reply:
New idea for `mode`, probably change the name and allow users to input a symbolic list e.g. ["r", "g", "b"] or a list of wavelength values e.g. [720, 560, 480].